pkg/cmd/cli/bug/bug.go

```diff
@@ -0,0 +1,204 @@
+* ` + "`ark restore logs <restorename>`" + `
+
+
+**Anything else you would like to add:**
+[Miscellaneous information that will assist in solving the issue.]
+
+
+**Environment:**
+
+- Ark version (use ` + "`ark version`" + `):{{.ArkVersion}} {{.GitCommit}} {{.GitTreeState}}
+- Kubernetes version (use ` + "`kubectl version`" + `): 
+{{- if .KubectlVersion}}
+` + "```" + `
+{{.KubectlVersion}}
+` + "```" + `
+{{end}}
+- Kubernetes installer & version:
+- Cloud provider or hardware configuration:
+- OS (e.g. from ` + "`/etc/os-release`" + `):
+{{if .RuntimeOS}}	- RuntimeOS: {{.RuntimeOS}}{{end -}}
+{{if .RuntimeArch}}	- RuntimeArch: {{.RuntimeArch}}{{end -}}
+`
+)
+
+func NewCommand() *cobra.Command {
+	c := &cobra.Command{
+		Use:   "bug",
+		Short: "Report an Ark bug",
+		Long:  "Open a browser window to report an Ark bug",
+		Run: func(c *cobra.Command, args []string) {
+			kubectlVersion, err := getKubectlVersion()
+			if err != nil {
+				// we don't want to prevent the user from submitting a bug
+				// if we can't get the kubectl version, so just display a warning
+				fmt.Fprintf(os.Stderr, "WARNING: can't get kubectl version: %v\n", err)
+			}
+			body, err := renderToString(newBugInfo(kubectlVersion))
+			cmd.CheckError(err)
+			cmd.CheckError(showIssueInBrowser(body))
+		},
+	}
+	return c
+}
+
+type ArkBugInfo struct {
+	ArkVersion     string
+	GitCommit      string
+	GitTreeState   string
+	RuntimeOS      string
+	RuntimeArch    string
+	KubectlVersion string
+}
+
+// cmdExistsOnPath checks to see if an executable is available on the current PATH
+func cmdExistsOnPath(name string) bool {
+	if _, err := exec.LookPath(name); err != nil {
+		return false
+	}
+	return true
+}
+
+// getKubectlVersion makes a best-effort to run `kubectl version`
+// and return it's output. This func will timeout and return an empty
+// string after kubectlTimeout if we're not connected to a cluster.
+func getKubectlVersion() (string, error) {
+	if !cmdExistsOnPath("kubectl") {
+		return "", errors.New("kubectl not found on PATH")
+	}
+
+	kubectlCmd := exec.Command("kubectl", "version")
+	var outbuf bytes.Buffer
+	kubectlCmd.Stdout = &outbuf
+	if err := kubectlCmd.Start(); err != nil {
+		return "", errors.New("can't start kubectl")
+	}
+
+	done := make(chan error, 1)
+	go func() {
+		done <- kubectlCmd.Wait()
+	}()
+	select {
+	case <-time.After(kubectlTimeout):
+		// we don't care about the possible error returned from Kill() here,
+		// just return an empty string
+		kubectlCmd.Process.Kill()
+		return "", errors.New("timeout waiting for kubectl version")
+
+	case err := <-done:
+		if err != nil {
+			return "", errors.New("error waiting for kubectl process")
+		}
+	}
+	versionOut := outbuf.String()
+	kubectlVersion := strings.TrimSpace(string(versionOut))
+	return kubectlVersion, nil
+}
+
+func newBugInfo(kubectlVersion string) *ArkBugInfo {
+	bugInfo := ArkBugInfo{
```

Comment:
Shoot, I forgot to add this comment: I'm pretty sure this variable here is redundant, you may `s/bugInfo :=/return`.

Reply:
good catch, fixed!